quarkus-debezium-parent/quarkus-debezium-testsuite-parent/deployment/src/test/java/io/quarkus/debezium/testsuite/deployment/TestSuiteConfigurations.java

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.quarkus.debezium.testsuite.deployment;
+
+public class TestSuiteConfigurations {
+
+    public static final int TIMEOUT = Integer.parseInt(System.getProperty("QUARKUS.TESTSUITE.TIMEOUT", "100"));
```

Comment:
Could we use lowercase for the system property name, as that aligns with most of our other configuration properties, which we typically define based on system properties?